paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/USBankAccountFormViewModel.kt

```diff
@@ -526,6 +526,15 @@ internal class USBankAccountFormViewModel @Inject internal constructor(
             amount = args.formArgs.amount?.value,
             currency = args.formArgs.amount?.currencyCode,
             linkMode = args.linkMode,
+            prefillDetails = makePrefillDetails(),
+        )
+    }
+
+    private fun makePrefillDetails(): ElementsSessionContext.PrefillDetails {
+        return ElementsSessionContext.PrefillDetails(
+            email = email.value ?: defaultBillingDetails?.email,
```

Comment:
We want to fall back to any billing details that the merchant has passed in, even if it’s not being attached to the `PaymentMethod`.